python2/koans/about_exceptions.py

```diff
@@ -65,4 +65,4 @@ def test_finally_clause(self):
         finally:
             result = 'always run'
 
-        self.assertEqual(__, result)
+        self.assertEqual('always run', result)
```

Comment:
In your own words, when do the `else` and `finally` blocks get executed?

Reply:
The `else` is executed if the `try` doesn't catch any exception, while the `finally` on the other hand is always runned at the end of the execution, no matter what is the outcome of the `try-except` is.